### PR TITLE
Enable skipping slow tests.

### PR DESCRIPTION
**Pull request by karandesai-96**
Certain tests can be skipped in normal run of tests and would be executed only on the provision of `--run-slow` parameter. This means:
* `./setup.py test` will skip all slow marked tests inherently.
* `./setup.py test --slow` will include all the tests while running.

### CODE REVIEW

**Comment by karandesai-96** [comment]
@wkerzendorf Just as we decided, a separate PR which _just_ adds the skip marker in **conftest.py**. I will be basing my next PR to add the integration test for stratified W7 setup on this branch itself. I will be doing the rebase to **master** once this is merged.

File: tardis/conftest.py [hunk 1]
```diff
@@ -32,6 +32,10 @@ def pytest_addoption(parser):
     parser.addoption("--atomic-dataset", dest='atomic-dataset', default=None,
                      help="filename for atomic dataset")
 
+    parser.addoption("--with-slow", action="store_true",
```
Comment (wkerzendorf) [inline tardis/conftest.py]: just change this to `--slow`. otherwise it looks good.

Comment (karandesai-96) [inline tardis/conftest.py]: - [X] Modified

**Comment by wkerzendorf** [comment]
@yeganer - I'm fine to merge this one.

**Comment by unoebauer** [comment]
Easy enough - looks good - merging

**Pull request merged**